src/tests/spec/corpus.rs

```diff
@@ -61,63 +61,95 @@ fn run_test(test: TestFile) {
 
         let canonical_bson = hex::decode(&valid.canonical_bson).expect(&description);
 
-        let bson_to_native_cb =
```

Comment:
I went ahead and updated the naming scheme of the different things here. Let me know if this makes sense. I know the old one got the point of being unreadable to me at least, so hopefully this is better.